frank-doc-doclet/src/main/java/org/frankframework/frankdoc/wrapper/FrankClassDoclet.java

```diff
@@ -64,6 +65,19 @@ class FrankClassDoclet implements FrankClass {
 		}
 		AnnotationDesc[] annotationDescs = clazz.annotations();
 		frankAnnotationsByName = FrankDocletUtils.getFrankAnnotationsByName(annotationDescs);
+		initializeVariables();
+	}
+
+	private void initializeVariables() {
+		for(FieldDoc fieldDoc: clazz.fields()) {
+			Object value = fieldDoc.constantValue();
+			if(value != null) {
+				variables.put(fieldDoc.name(), value.toString());
+			}
+		}
+		for(FieldDoc fieldDoc: clazz.enumConstants()) {
+			variables.put(fieldDoc.name(), fieldDoc.name());
+		}
```

Comment:
Het label is er juist om als waarde te fungeren in plaats van de naam. Je moet dus als er een label is het label gebruiken.

Reply:
Ik heb erin gestopt dat het label wordt genomen als er een @EnumLabel annotatie is. Ik heb gecontroleerd dat FrankConfig.xsd door deze update niet verandert voor de huidige F!F sources.